src-docs/src/views/suggest/saved_queries.js

```diff
@@ -123,6 +123,7 @@ export default () => {
             onBlur={onFieldBlur}
             prepend={<HashtagPopover value={value} />}
             append={append}
+            aria-label="EuiSuggest input field"
```

Comment:
Similar to above, don't need to specify what it is - the screen reader will handle that. We should specify what it does.

```suggestion
            aria-label="Filter"
```